Postgresql/legos/postgresql_read_query/postgresql_read_query.py

```diff
@@ -60,8 +60,13 @@ def postgresql_read_query(handle, query: str, params: tuple = ()) -> List:
         prepared_query = "EXECUTE psycop_{random_id};".format(
             random_id=random_id)
     else:
+        parameters_tuple = tuple(params)
+        ## If there is only one tuple element, remove the trailing comma before format
+        if len(parameters_tuple) == 1:
+            tuple_string = str(parameters_tuple)
+            parameters_tuple = tuple_string[:-2] + tuple_string[-1]
```

Comment:
What will this command do? In the comments take an example and show how the tuple would look like whe. There is single parameter

Reply:
A tuple with a single element looks like ("foo",). We need ("foo"). Adding this in comments